tangelo/linq/simulator.py

```diff
@@ -26,515 +26,64 @@
 (e.g a number of shots is required).
 
 Some backends may only support a subset of the above. This information is
-contained in a separate data-structure.
+contained in the static method backend_info for each backend.
 """
 
-import os
-import math
-from collections import Counter
+from typing import Union, Tuple, Type
 
-import numpy as np
-from scipy import stats
-from bitarray import bitarray
-from openfermion.ops import QubitOperator
+import numpy
```

Comment:
I don't believe your code needs this anymore. Does your IDE highlight unused packages? I think this is not the first time we come across that.

Reply:
numpy was used for a typing but I realized that `simulate_circuit` is really a local method and the typing can be provided by `simulate`.